guides/source/active_record_multiple_databases.md

```diff
@@ -102,6 +102,9 @@ Lastly, for new writer databases, you need to set the `migrations_paths` to the
 where you will store migrations for that database. We'll look more at `migrations_paths`
 later on in this guide.
 
+You can also configure the schema dump file by setting `schema_dump` to a custom schema file name
+or completely skip the schema dumping by setting `schema_dump: false`.
```

Comment:
Maybe we could add a link to https://edgeguides.rubyonrails.org/active_record_migrations.html#schema-dumping-and-you

Reply:
I honestly do not think that this is needed. It is expected, that people know what it is 🤔 .

Reply:
I thought so too, probably safe to assume if they are setting up multiple dbs they already are familiar with that kind of stuff. Just thought I'd ask anyway :pray: Thanks for working on this!